clients/vscode-hlasmplugin/scripts/syntaxes/hlasm_tmgrammar_template.txt

```diff
@@ -400,8 +400,20 @@
       }
     },
     "remark": {
-      "match": "(\\S+)(?<=^${listingDetails}$.{0,71})",
-      "name": "comment.hlasm"
+      "patterns": [
+        {
+          "match": "^${listingDetails}$\\s{15}(.*)(?<=^${listingDetails}$.{0,71})",
```

Comment:
Doesn't this interfere with the simple operand continuation?

Reply:
Do you mean this

```
LABEL INSTR OP1,OP2,                                                   -12346578
               OP3
```


or this

```
LABEL INSTR                                                        OPER-12346578
               AND
```
?


The rule is now (slightly) changes, however, in either case, it looks fine:
![image](https://github.com/BroadcomMFD-DevOps/che-che4z-lsp-for-hlasm/assets/99467904/13996c78-11a9-43ed-bbd3-ed0ae9b6a371)


However, this specific test case seems to be missing so I might add it later